File: easy/fizzbuzz/go/fizzbuzz.go
package main

import "fmt"
import "log"
import "bufio"
import "os"
import "strings"
import "strconv"

func main() {
    file, err := os.Open(os.Args[1])
    if err != nil {
        log.Fatal(err)
    }   
    defer file.Close()
    scanner := bufio.NewScanner(file)
    for scanner.Scan() {
        values := strings.Split(scanner.Text(), " ")
        firstDivisor, _ := strconv.Atoi(values[0])
        secondDivisor, _ := strconv.Atoi(values[1])
        n, _ := strconv.Atoi(values[2])
        fizzbuzz(firstDivisor, secondDivisor, n)
    }
}

func fizzbuzz(firstDivisor int, secondDivisor int, n int) {
    for i := 1; i <= n; i++ {
        output := ""
        if i % firstDivisor == 0 {
            output += "F"
        }
        if i % secondDivisor == 0 {
            output += "B"
        }
        if output == "" {
            output += strconv.Itoa(i)
        }

        fmt.Print(output + " ")
    }

    fmt.Println()
}

File: easy/fizzbuzz/ruby/fizzbuzz.rb
def fizzbuzz(firstDivisor, secondDivisor, n)
	n.times do |currentInteger|
		currentInteger = currentInteger + 1
		output = ""
		firstRemainder = currentInteger % firstDivisor
		secondRemainder = currentInteger % secondDivisor

		if firstRemainder == 0
			output += "F"
		end

		if secondRemainder == 0
			output += "B"
		end

		if output == ""
			output = currentInteger.to_s
		end

		print output + " "
	end

	puts
end

File.open(ARGV[0]).each_line do |line|
	values = line.split(" ")
	fizzbuzz values[0].to_i, values[1].to_i, values[2].to_i
end